src/sdi-refresh-monitor.c

```diff
@@ -440,8 +445,15 @@ static void manage_change_update(SnapdClient *source, GAsyncResult *res,
   if (change == NULL)
     return;
 
-  gboolean done = g_str_equal(snapd_change_get_status(change), "Done");
-  gboolean cancelled = cancelled_change_status(snapd_change_get_status(change));
+  const gchar *change_status = snapd_change_get_status(change);
+
+  gboolean done = g_str_equal(change_status, "Done");
+  gboolean cancelled = cancelled_change_status(change_status);
+  gboolean valid_do = valid_working_change_status(change_status);
+  if (!(valid_do || cancelled)) {
+    g_debug("Unknown change status %s\n", change_status);
```

Comment:
Don't need the newline at the end - g_debug adds this already.

Reply:
Fixed.